src/index.ts

```diff
@@ -1,11 +1,9 @@
 import { EventEmitter } from 'eventemitter3'
-import Promise from 'ts-promise'
```

Comment:
Let's verify this still works in a non-promise browser. Seems like TS is building a promise shim in, but want to make sure.

Reply:
I don't have a local IE to test it on. Suggestions?